client/src/main/scala/caliban/client/Value.scala

```diff
@@ -8,47 +8,47 @@ import io.circe.{ Decoder, Encoder, Json }
 sealed trait Value
```

Comment:
Is it better to rename `Value` to `__Value`?
I think the type may have the potential to collide with a user type name.

Reply:
Yeah, I agree.